Benchmark-Models/Blasi_CellSystems2016/petab_select/input/petab_problem.yaml

```diff
@@ -0,0 +1,11 @@
+format_version: 1
+parameter_file: parameters.tsv
```

Comment:
@dilpath  shall this adapt the conventional naming as for the rest of the models? e.g. https://github.com/Benchmarking-Initiative/Benchmark-Models-PEtab/tree/blasi_model_selection/Benchmark-Models/Blasi_CellSystems2016
(for all files here)

Reply:
After working with PEtab for a while, it's my preference. There is a helper method (`to_files_generic`) in recent versions of PEtab that makes this easy [1] (i.e. could loop over models in the repository, read PEtab problems, then write with this helper method).

However, this may break the code of any user of this collection, if it relies on the current naming convention (e.g., `PROBLEM_ID/PROBLEM_ID.yaml` instead of `PROBLEM_ID/petab_problem.yaml`).

The reason why this naming is used in this PR is only because the `to_files_generic` method is used, not because of a change in the naming convention. I think this doesn't contradict the current naming in the rest of the benchmark collection, since this PEtab (Select) problem is specifically for the model selection problem and is not intended to be used alongside the other benchmark collection models.

It might be more appropriate to share these PEtab Select problems as a separate collection, but also useful here I think, so any user of the model can see there is this alternative model and PEtab problem formulation.

[1] https://github.com/PEtab-dev/libpetab-python/blob/6e2a7c7136c7e161b71cda72cdbf7756535e8d22/petab/problem.py#L298